docs/element-definitions.md

```diff
@@ -424,15 +474,15 @@ Type with domain of all rational numbers, i.e. floating-point numbers with finit
 - `element` - `"number"`
 - `attributes`
   - `typeAttributes` ([Array](#array-element)[[String](#string-element)])
-    - `fixed` ([String](#string-element)) - domain restricted to value given in `content`
+    - `fixed` ([String](#string-element)) - The type this Element describes is restricted to the value given in `content`
   - `validation` - _reserved for future use_
-  - `samples` ([Array[Number]](#array-element)) - array of alternative sample values for this Element; type of items in `samples` MUST match the type of this Element
-  - `default` (Number) - default value for this Element; type of `default` MUST match the type of this Element
-- `content` - rational number
+  - `samples` ([Array[[Number](#number-element)]](#array-element)) - Alternative sample values for this Element; type of items in `samples` MUST match the type this Element describes
```

Comment:
Bad linking. Use something similar to `([Array](#array-element)[[String](#string-element)])`